knowledge_storm/collaborative_storm/modules/warmstart_hierarchical_chat.py

```diff
@@ -0,0 +1,398 @@
+import dspy

```

Comment:
Add a comment on top of this file as I don't think many people can understand what `warmstart_hierarchical_chat` means.